components/tikv_util/src/yatp_pool/future_pool.rs

```diff
@@ -165,7 +190,6 @@ impl PoolInner {
             return Ok(());
         }
 
-        let current_tasks = self.get_running_task_count();
         if current_tasks >= max_tasks {
             Err(Full {
```

Comment:
add priority?

Reply:
The content of `Full` type is not used currently, we can add in the future if new logic depends on the error detail.